obspy/taup/tau.py

```diff
@@ -663,3 +713,91 @@ def create_taup_model(model_name, output_dir, input_dir):
     else:
         model_file_name = model_name + ".tvel"
     TauPCreate.main(model_file_name, output_dir, input_dir)
+
+
+def traveltime_plot(min_degree=0, max_degree=360, npoints=1000,
+                    phases=['P', 'S'], source_depth=10, model='iasp91',
```

Comment:
I would vote for making `source_depth` an argument, not a `kwarg` with default value, such that people have to explicitly set their source depth of choice. Otherwise people might overlook it's importance for the output and I feel like there's no generally useful default value for it.